matchbox_socket/src/webrtc_socket/error.rs

```diff
@@ -45,15 +46,18 @@ pub enum SignalingError {
 }
 
 /// An error that can occur with WebRTC messaging.
-#[cfg(not(target_arch = "wasm32"))]
 #[derive(Debug, thiserror::Error)]
-#[error("failed to send message to peer: {0}")]
-pub(crate) struct MessagingError(#[from] futures_channel::mpsc::TrySendError<crate::Packet>);
+pub enum MessageSendError {
+    #[error("failed to send peer id to recipient: {0}")]
+    PeerId(PeerId),
```

Comment:
*if* we keep the error, I think it should make it more clear that this is that we received a peer id after the socket was dropped, and not something outgoing.

Reply:
error removed